Dockerfile

```diff
@@ -12,12 +12,16 @@ RUN apt-get update -y && \
 ADD config/nginx.conf /opt/etc/nginx.conf
 RUN rm /etc/nginx/sites-enabled/default
 
+#Declaring the data dir
+ENV data_root /data
+
 # Nginx startup script
 ADD config/nginx-start.sh /opt/bin/nginx-start.sh
 RUN chmod u=rwx /opt/bin/nginx-start.sh
 
-RUN mkdir -p /data
-VOLUME ["/data"]
+RUN mkdir -p ${data_root}
```

Comment:
@roshangn88 Can you format these variables as `${data_root:-/data}` so that the if the `data_root` env var is not set, then there is a default.

See https://docs.docker.com/reference/builder/#environment-replacement for more info